### ISSUE TITLE
Encrypted Drive

### ISSUE THREAD

**Issue by JoKerIsCraZy** [m-rots/a-train#3]
Hey is it possible to add a encrypted google drive? (encrypted with rclone)

**Comment by herodev1337** [comment]
I would also be very interested in solving this problem

**Comment by hongkongkiwi** [comment]
Yup, this seems key for many users. Perhaps A-Train could support rclone's encryption support and that way rclone can handle the heavy lifting?

**Comment by gurabli** [comment]
I also miss rclone encrypted (Shared Drive) support. Any way to add this?
